Automattic/jetpack
# Boost: Add first user flow

Fixes 142-gh-Automattic/boost-cloud

#### Changes proposed in this Pull Request:

- [x] Added a pricing table page that compares between the free and paid plan of boost
- [x] User can continue to the purchase flow by clicking the purchase button on the new page
- [x] The purchase page will automatically appear for first-time users
- [x] Once a user chooses a plan, the purchase plan will no longer appear
- [x] Rewrite the pricing component in svelte
- [x] Plug in pricing from real data source

#### Other information:


#### Jetpack product discussion
p1HpG7-hBQ-p2 

#### Does this pull request change what data or activity we track or use?
No



#### Testing instructions:
* For now, you need to manually access the page from: `wp-admin/admin.php?page=jetpack-boost#/getting-started`
* Set option `jb_get_started` to true to force the user to redirect to the get started page every time they access boost dashboard.
* Choose a plan, and the forced redirect to the getting started page should go away.

##### Notes
- The CTA on free plan will permanently disable the getting started page.
- Choosing the paid plan will not disable the getting started page, it will only redirect to the purchase flow.
- Accessing the purchase success page(`wp-admin/admin.php?page=jetpack-boost#/purchase-successful` will disable the getting started page.
